README.md

```diff
@@ -16,7 +16,15 @@ If you want to change anything in the CSS,
   and there run the command `npm install`
 - "Compilation" (mostly of the Less sources): Run the command
   `npm run build`
-  
+
+
+To create the speakers page you need to run the speakers.py script from
+inside the virtualenv like this (assuming the site is generated from
+website_2024 folder):
+
+    python ./speakers.py <event-name> -t <api-token> -o ./website_2024/content/pages/speakers.md
+
+
 When you want to build the site itself, use `make html` from the
```

Comment:
Hi,

That's good in what it includes, but it's missing two parts: One is that generating the
speaker's page is an essential part of building the site, and the other is where the
script parameters come from.

So:

```suggestion

When you want to build the site itself, there's two things you need to do:

First, to create the speakers page, you need to run the 
`speakers.py` script (found in the repo's root folder) from
inside the virtualenv like this (assuming the site is generated from
website_2024 folder):

    python ./speakers.py <event-slug> -t <api-token> -o ./website_2024/content/pages/speakers.md

In this command, `<event-slug>` is the slug for the event in Pretalx,
and `<api-token>` is an authentication token you can get from your 
Pretalx profile page. For PyCon Israel 2024, `event-slug` is `pycon-2024`,
and your profile page is https://cfp.pycon.org.il/pycon-2024/me/ .

Then, use `make html` from the
```